src/auth/index.ts

```diff
@@ -122,6 +122,15 @@ export {
   TenantManager,
 } from './tenant-manager';
 
+export {
+  UpdateProjectConfigRequest,
```

Comment:
Should we expose GetProjectConfigRequest here?

Reply:
We don't have a getProjectConfigRequest.  It takes an empty object.